### ISSUE TITLE
Header dropdown open state can be backwards

### ISSUE THREAD

**Issue by kr-matthews** [thewca/wca-components#1]
If you hover on a dropdown (which causes the dropdown to open) then click it, it closes. And moving the cursor away makes it open; and moving back over it closes it.

I'm not sure exactly what's causing this, but [these](https://github.com/thewca/wca-components/blob/154bc0036f2d367727507e1be8a291057200a092/src/components/Header/Dropdown.tsx#LL38C2-L38C2) could explicitly open/close, rather than toggle.
```
      onMouseEnter={toggleHover}
      onMouseLeave={toggleHover}
```

**Comment by FinnIckler** [comment]
Yeah that should fix it. This is caused by me trying to replicate the way it currently works on the website, where clicking on it also closes it

**Comment by FinnIckler** [comment]
Fixed in Version 0.2.3
